tests/performance/locust_files/catalog_servicesd.py

```diff
@@ -0,0 +1,60 @@
+#
+# SEE https://docs.locust.io/en/stable/quickstart.html
+#
+
```

Comment:
MINOR: why ending in "*d.py" the filename?

Reply:
oups... d for direct isn't that obvious? :rofl: will change it